### ISSUE TITLE
Are the output layers concatenated properly ?

### ISSUE THREAD

**Issue by paniabhisek** [Currie32/Text-Summarization-with-Amazon-Reviews#8]
Here is the `encoding_layer` for ease:
```
def encoding_layer(rnn_size, sequence_length, num_layers, rnn_inputs, keep_prob):
    '''Create the encoding layer'''
    
    for layer in range(num_layers):
        with tf.variable_scope('encoder_{}'.format(layer)):
            cell_fw = tf.contrib.rnn.LSTMCell(rnn_size,
                                              initializer=tf.random_uniform_initializer(-0.1, 0.1, seed=2))
            cell_fw = tf.contrib.rnn.DropoutWrapper(cell_fw, 
                                                    input_keep_prob = keep_prob)

            cell_bw = tf.contrib.rnn.LSTMCell(rnn_size,
                                              initializer=tf.random_uniform_initializer(-0.1, 0.1, seed=2))
            cell_bw = tf.contrib.rnn.DropoutWrapper(cell_bw, 
                                                    input_keep_prob = keep_prob)

            enc_output, enc_state = tf.nn.bidirectional_dynamic_rnn(cell_fw, 
                                                                    cell_bw, 
                                                                    rnn_inputs,
                                                                    sequence_length,
                                                                    dtype=tf.float32)
    # Join outputs since we are using a bidirectional RNN
    enc_output = tf.concat(enc_output,2)
    
    return enc_output, enc_state
```

I'm new to `tensorflow`, so here's my understanding:
Based on `num_layers`, it'll create that many `tf.nn.bidirectional_dynamic_rnn`s. So shouldn't the line `enc_output = tf.concat(enc_output,2)` be inside the for loop ? Will the return statement returns all the `enc_output`. If so how ?

And how to find accuracy, can you suggest briefly ?
